conf/update.routes

```diff
@@ -83,4 +83,4 @@ GET        /:operatorId/updated-successfully                controllers.update.P
 GET        /remove/:operatorId/are-you-sure                 controllers.update.RemovePlatformOperatorController.onPageLoad(operatorId: String)
 POST       /remove/:operatorId/are-you-sure                 controllers.update.RemovePlatformOperatorController.onSubmit(operatorId: String)
 
-GET        /remove/:operatorId/removed-successfully         controllers.update.PlatformOperatorRemovedController.onPageLoad(operatorId: String)
+GET        /remove/:operatorId/removed-successfully         controllers.update.PlatformOperatorRemovedController.onPageLoad(operatorId: String, businessName: String)
```

Comment:
We shouldn't include businessName in the URL, as it's sensitive information and would be logged.  Instead, save the business name to user answers at the point of deletion (in a new Query) so we can read it back out on this page